pkg/chartvalues/apiextensions_chart_e2e.go

```diff
@@ -0,0 +1,75 @@
+package chartvalues
+
+import (
+	"github.com/giantswarm/e2etemplates/internal/render"
+	"github.com/giantswarm/microerror"
+)
+
+type APIExtensionsChartE2EConfig struct {
+	Chart         APIExtensionsChartE2EConfigChart
+	ChartOperator APIExtensionsChartE2EConfigChartOperator
+	ConfigMap     APIExtensionsChartE2EConfigConfigMap
+	Namespace     string
+	Secret        APIExtensionsChartE2EConfigSecret
+}
+
+type APIExtensionsChartE2EConfigChart struct {
+	Config     APIExtensionsChartE2EConfigChartConfig
+	Name       string
+	Namespace  string
+	TarballURL string
+}
+
+type APIExtensionsChartE2EConfigChartConfig struct {
+	ConfigMap APIExtensionsChartE2EConfigChartConfigConfigMap
+	Secret    APIExtensionsChartE2EConfigChartConfigSecret
+}
+
+type APIExtensionsChartE2EConfigChartConfigConfigMap struct {
+	Name      string
+	Namespace string
+}
+
+type APIExtensionsChartE2EConfigChartConfigSecret struct {
+	Name      string
+	Namespace string
+}
+
+type APIExtensionsChartE2EConfigChartOperator struct {
+	Version string
+}
+
+type APIExtensionsChartE2EConfigConfigMap struct {
+	ValuesYAML string
```

Comment:
We need to be able to override the configmap values for the e2e tests. But I don't think using a `map[string]interface{}` type helps here. 

So I called the struct variable `ValuesYAML` and it can be set in the e2e test.

Reply:
Yes this is ugly. Let's see how it goes. Maybe we can come up with something clever when use-cases are clearer.